### PR TITLE
BAM-3800: API Docs Requirements Updates

### PR DESCRIPTION
**Pull request by nickcroce**
Patch Thursday

### CODE REVIEW

File: conf/requirements.txt [hunk 1]
```diff
@@ -171,3 +174,7 @@ urllib3==1.25.3 \
 watchdog==0.9.0 \
     --hash=sha256:965f658d0732de3188211932aeb0bb457587f04f63ab4c1e33eab878e9de961d \
     # via sphinx-autobuild
+
+# WARNING: The following packages were not pinned, but pip requires them to be
+# pinned when the requirements file includes hashes. Consider using the --allow-unsafe flag.
+# setuptools==41.0.1        # via sphinx
```
Comment (csdev) [inline conf/requirements.txt]: what's going on here?

File: conf/requirements.txt [hunk 2]
```diff
@@ -12,13 +12,17 @@ argh==0.26.2 \
...
 babel==2.7.0 \
     --hash=sha256:af92e6106cb7c55286b25b38ad7695f8b4efb36a90ba483d7f7a6628c46158ab \
     --hash=sha256:e86135ae101e31e2c8ec20a4e0c5220f4eed12487d5cf3f78be7e98d3a57fc28 \
     # via sphinx
-certifi==2019.3.9 \
-    --hash=sha256:59b7658e26ca9c7339e00f8f4636cdfe59d34fa37b9b04f6f9e9926b3cece1a5 \
-    --hash=sha256:b26104d6835d1f5e49452a26eb2ff87fe7090b89dfcaee5ea2212697e1e1d7ae \
+certifi==2019.6.16 \
+    --hash=sha256:046832c04d4e752f37383b628bc601a7ea7211496b4638f6514d0e5b9acc4939 \
+    --hash=sha256:945e3ba63a0b9f577b1395204e13c3a231f9bc0223888be653286534e5873695 \
```
Comment (nickcroce) [inline conf/requirements.txt]: this ones a pretty big jump

File: conf/requirements.txt (hunk 1 above)
Comment (nickcroce) [reply conf/requirements.txt]: No idea, but I ran the local preview and everything is working as expected

Comment (nickcroce) [reply conf/requirements.txt]: No idea, but I ran the local preview and everything is working as expected

Comment (csdev) [reply conf/requirements.txt]: this happened after running `make pip-upgrade`? I'll need to investigate further

Comment (nickcroce) [reply conf/requirements.txt]: Yes

**Pull request merged**